src/p_saveg.c

```diff
@@ -1232,7 +1224,16 @@ static void saveg_read_vldoor_t(vldoor_t *str)
     str->topcountdown = saveg_read32();
 
     // line_t *line;
```

Comment:
Can we get rid of `readp()` altogether?

Reply:
Yes, we can explicitly skip pointers. But sometimes `NULL` is written with checks in other places, we have to find them all.

I suggest we hold off on this for now.

Reply:
Yes, okay.